src/network/ConnectionForward.ts

```diff
@@ -23,7 +23,6 @@ interface ConnectionForward extends StartStop {}
 @StartStop()
 class ConnectionForward extends Connection {
   public readonly nodeId: NodeId;
-  public readonly endTime: number;
```

Comment:
What's happening here? The `endTime` is still being used in a number of functions.

Reply:
Oh right this is because it's already available in `Connection.ts`. This should be extracted to a separate commit to make it clearer.

Reply:
it's defined in Connection, no need to redefine it there.

Reply:
Ok other priorities are becoming more important, resolving this for now.